compare.go

```diff
@@ -269,11 +269,11 @@ func CompareTransforms(a, b *Transform) error {
 	if a.Syntax != b.Syntax {
 		return fmt.Errorf("Syntax: %s != %s", a.Syntax, b.Syntax)
 	}
-	if a.AppVersion != b.AppVersion {
-		return fmt.Errorf("AppVersion: %s != %s", a.AppVersion, b.AppVersion)
+	if a.SyntaxVersion != b.SyntaxVersion {
```

Comment:
The only alternative is to use LanguageVersion, or LangVersion, but SyntaxVersion is good too.